README.md

```diff
@@ -1,12 +1,23 @@
-# barman-metrics
-Metrics exporter for barman 
+# cached-metrics
+Metrics exporter for Prometheus. 
 
-A small Flask app, designed to be run inside a docker container, that exposes a single `/metrics` endpoint returning
- metadata from the local barman instance in a format that Prometheus can use for monitoring.
+A small Flask app, designed to be run inside a docker container, that exposes a single `/metrics` endpoint 
+returning metadata from a local cache file in a format that Prometheus can use for monitoring. 
 
-Start by running:
+Clone with `--recursive` and start by running:
 
 ```
 sudo -H pip3 install -r requirements.txt
-./run
+sudo -H pip3 install -r ./montagu_metrics/requirements.txt
+./scripts/run
 ```
+
+## Tests
+
+To run tests inside a docker container, with `config-example.json` mounted as the config file:
+```
+./scripts/teamcity.sh
```

Comment:
The teamcity build built for me a different container than the one that `run` built - is that expected?